app/scripts/services/wsService.js

```diff
@@ -143,6 +143,14 @@ wsService.factory('dbXrefService', ['$http', '$location', function($http, $locat
   };
 }])
 
+wsService.factory('olsService', ['$http', '$location', function($http, $location) {
+  return {
+    getTermName: function(xref) {
+      return $http.get('http://www.ebi.ac.uk/ols/api/ontologies/' + xref.db.toLowerCase() + '/terms/' + 'http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252F' + xref.db + '_' + xref.id);
```

Comment:
Are we sure all of the OLS DBs used in GO will also use http://purl.obolibrary.org/obo/ as IRI? Because that is not the case for all the ontologies supported by OLS